project_task_send_by_mail/views/project_task_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <record id="view_task_form2" model="ir.ui.view">
+        <field name="name">project.task.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <field name="stage_id" position="before">
+                <button name="action_task_send" type="object" string="Send by Email" groups="base.group_user"/>
```

Comment:
Why the group here?

Reply:
@tarteo I did not add this group, It's already v11